python/pfs/drp/stella/fitPfsFluxReference.py

```diff
@@ -342,16 +343,22 @@ def getRadialVelocities(self, pfsConfig, pfsMerged, pfsMergedLsf, bbPdfs):
         # This model is used as the reference for cross-correlation calculation
         bestModels = self.findRoughlyBestModel(bbPdfs)
 
-        radialVelocities = []
-        for iFiber, (spectrum, model) in enumerate(zip(fibers(pfsConfig, pfsMerged), bestModels)):
-            if model.spectrum is None:
-                radialVelocities.append(None)
-                continue
-            modelSpectrum = convolveLsf(
-                model.spectrum, pfsMergedLsf[pfsConfig.fiberId[iFiber]], spectrum.wavelength
-            )
-            modelSpectrum = self.computeContinuum(modelSpectrum, mode="model").whiten(modelSpectrum)
-            radialVelocities.append(self.estimateRadialVelocity.run(spectrum, modelSpectrum))
+        originalMask = self.estimateRadialVelocity.config.mask
+        self.estimateRadialVelocity.config.mask = self.getBadMask(mask=originalMask)
```

Comment:
How should I replace the default config of a subtask (`estimateRadialVelocity`) with another one? I don't think it good to set the default config itself to `[..., "ATOMOSPHERE", "EDGE"]` because these two flags are used only temporarily by `fitPfsFluxReference` and are not expected by `estimateRadialVelocity`.

Reply:
Maybe `setDefaults()`